mx.py

```diff
@@ -3625,14 +3625,14 @@ def build(self):
                 for fname in filenames:
                     output.append(os.path.join(root, fname))
             if output:
-                key = lambda x: os.path.getmtime(_safe_path(x))
+                key = lambda x: getmtime(x)
```

Comment:
This is causing warnings in pylint. You can remove that local variable and use `key=getmtime` below.